### PR TITLE
Update references to Expo / Expo CLI

### PR DESCRIPTION
**Pull request by brentvatne**
We recently made some changes, [such as moving away from the global Expo CLI package to a local CLI and `create-expo-app`](https://blog.expo.dev/the-new-expo-cli-f4250d8e3421) and [moving away from `expo build`](https://blog.expo.dev/turtle-goes-out-to-sea-d334db2a6b60). 

This pull requests accounts for these changes and backports them to the 0.69 docs, because Expo SDK 46 currently uses React Native 0.69.

### CODE REVIEW

**Review by cortinico — APPROVED** [review]
LGTM 👍 Just a minor comment left

Also are there any implications for having stale global `expo-cli` packages?

File: docs/getting-started.md [hunk 1]
```diff
@@ -10,52 +10,33 @@ import GuideLinuxAndroid from './\_getting-started-linux-android.md'; import Gui
 
 This page will help you install and build your first React Native app.
 
-**If you are new to mobile development**, the easiest way to get started is with Expo CLI. Expo is a set of tools built around React Native and, while it has many [features](https://expo.io/features), the most relevant feature for us right now is that it can get you writing a React Native app within minutes. You will only need a recent version of Node.js and a phone or emulator. If you'd like to try out React Native directly in your web browser before installing any tools, you can try out [Snack](https://snack.expo.dev/).
+**If you are new to mobile development**, the easiest way to get started is with Expo Go. Expo is a set of tools and services built around React Native and, while it has many [features](https://docs.expo.dev), the most relevant feature for us right now is that it can get you writing a React Native app within minutes. You will only need a recent version of Node.js and a phone or emulator. If you'd like to try out React Native directly in your web browser before installing any tools, you can try out [Snack](https://snack.expo.dev/).
```
[the remark this answers is not in the document]
Comment (brentvatne) [reply docs/getting-started.md:13]: I think the best place for the notice is within the tool itself, so we added this note when you run a command with the deprecated expo-cli:

image

**Comment by brentvatne** [comment]
> Also are there any implications for having stale global expo-cli packages?

In order to ensure developers are using the correct CLI version we have added a warning (screenshotted in the other comment) to make it clear which CLI is being invoked. If a developer leaves the old expo-cli installed on their machine, it won't cause any problems for them and they may want to keep it around to use on older projects.

File: docs/getting-started.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (cortinico) [reply docs/getting-started.md:13]: That's great. Just note that if someone ⌘ + F for `expo-cli` in the website it won't find anything at this stage. I believe it's fine but just so you know 👍

**Pull request merged**